client/app/stores/router_store.coffee

```diff
@@ -186,7 +186,12 @@ class RouterStore extends Store
             @emit 'change'
 
         handle ActionTypes.ROUTE_CHANGE, (value) ->
-            _action = value
+            # We cant display any informations
+            # without accounts
+            unless AccountStore.getAll()?.size
+                _action = AccountActions.CREATE
```

Comment:
As long as `ROUTE_CHANGE` become the default event dispacthed by the router on url updates, are we sure that the requested action is `AccountActions.CREATE` or should we use the `value.action` passed by the router instead?